test/common/change_notifier_test_util.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:flutter/foundation.dart';
 
 extension ChangeNotifierTextExtensions on ChangeNotifier {
-  void verifyStateInOrder(
+  void verifyStateInOrder({
     Function() testFunction,
```

Comment:
named parameters bardzo spoko, ale jako że oba są teraz opcjonalne to dodałbym @required do nich :)